### DISCUSSION TITLE
Station range - option to include basestation and SAR aircraft in range

### DISCUSSION THREAD

**Discussion by flygfantast** [jvde-github/AIS-catcher#305] [Ideas]
Basestations and SAR aircraft seems to be excluded from the station range. I would like an option to include Basestation and SAR aircraft in the station range. Or even better have them as separate ranges.

I have many SAR helicopter in the area and sometimes I also see basestations far away.

**Comment by jvde-github** [comment]
Nice suggestion. Will look into it if it can be easily done. The complication is the backward compatibility with the backup file.
